docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+styleguide:
```

Comment:
Curious about if it makes sense to split the dev build and the ci build?  Do folks plan on using this docker image to do development on?

Reply:
That's a good question. Right now this image isn't really set up for development. The Jest step is baked into the Dockerfile itself.